powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/PowerAuthAuditConfiguration.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.getlime.security.powerauth.app.server.configuration;
+
+import com.wultra.core.audit.base.Audit;
+import com.wultra.core.audit.base.AuditFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration of auditing for PowerAuth.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Configuration
+@ComponentScan(basePackages = {"com.wultra.core.audit.base"})
```

Comment:
ComponentScan might be quite heavy. Let's discuss it later. I have opened an issue for me #726

Reply:
Let's keep this change outside of current release.